UnityProject/Assets/Scripts/Objects/BarSign.cs

```diff
@@ -59,11 +59,6 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 			return true;
 		}
 		
-		public void OnSpawnServer()
-		{
-			ChangeSign(signIndex);
```

Comment:
Unintentional?

Reply:
This is unused as barsign.cs doesnt have the IServerSpawn interface so I figured I might as well remove it